exchanges/okx/okx_types.go

```diff
@@ -2568,41 +3273,41 @@ type WsAdvancedAlgoOrder struct {
 
 // WsAdvancedAlgoOrderDetail advanced algo order response pushed through the websocket conn
 type WsAdvancedAlgoOrderDetail struct {
-	ActualPrice            string           `json:"actualPx"`
-	ActualSide             string           `json:"actualSide"`
-	ActualSize             string           `json:"actualSz"`
-	AlgoID                 string           `json:"algoId"`
-	Currency               string           `json:"ccy"`
-	Count                  string           `json:"count"`
-	InstrumentID           string           `json:"instId"`
-	InstrumentType         string           `json:"instType"`
-	Leverage               string           `json:"lever"`
-	NotionalUsd            string           `json:"notionalUsd"`
-	OrderPrice             string           `json:"ordPx"`
-	OrdType                string           `json:"ordType"`
-	PositionSide           string           `json:"posSide"`
-	PriceLimit             string           `json:"pxLimit"`
-	PriceSpread            string           `json:"pxSpread"`
-	PriceVariation         string           `json:"pxVar"`
-	Side                   order.Side       `json:"side"`
-	StopLossOrderPrice     string           `json:"slOrdPx"`
-	StopLossTriggerPrice   string           `json:"slTriggerPx"`
-	State                  string           `json:"state"`
-	Size                   string           `json:"sz"`
-	SizeLimit              string           `json:"szLimit"`
-	TradeMode              string           `json:"tdMode"`
-	TimeInterval           string           `json:"timeInterval"`
-	TakeProfitOrderPrice   string           `json:"tpOrdPx"`
-	TakeProfitTriggerPrice string           `json:"tpTriggerPx"`
-	Tag                    string           `json:"tag"`
-	TriggerPrice           string           `json:"triggerPx"`
-	CallbackRatio          string           `json:"callbackRatio"`
-	CallbackSpread         string           `json:"callbackSpread"`
-	ActivePrice            string           `json:"activePx"`
-	MoveTriggerPrice       string           `json:"moveTriggerPx"`
-	CreationTime           okxUnixMilliTime `json:"cTime"`
-	PushTime               okxUnixMilliTime `json:"pTime"`
-	TriggerTime            okxUnixMilliTime `json:"triggerTime"`
+	ActualPrice            string     `json:"actualPx"`
```

Comment:
across this type we can use types.Number can you please review all the structs here?